src/utils/conversion.ts

```diff
@@ -3362,3 +3363,36 @@ export function agentGroupToPermissionGroup(
     }
   }
 }
+
+/**
+ * @hidden
+ */
+export async function portfolioNameToNumber(
```

Comment:
Feels weird to have this as a conversion util, since it's not actually converting data, but rather fetching. I'd move it to `src/utils/internal.ts` and rename it to `getPortfolioIdByName`